src/token-adapter-svc/sdkRoutes.ts

```diff
@@ -74,4 +90,44 @@ export class SDKRoutes implements IRoutes {
         }
     }
 
+    //@ts-expect-error h has no type
+    private async postQuotes(request, h){
+        console.log("Received request: POST /quoterequests");
+
+        const payload: IQuoteRequestData = request.payload as IQuoteRequestData;
+
+        if(!payload.to){
+            return h.response("Bad Request: Payload missing crucial info").code(400);
+        }
+
+        const result = await this.sdkAggregate.postQuotes(payload);
+        if(!result){
+            return h.response({statusCode:3204,message:"Party not found"}).code(404);
```

Comment:
I think this error message should be more specific. Like alias not found or matched...etc